.unimportedrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This is the config I'm using, the ignore patterns are coming from the `npx unimported --init` commands.

@daawascript for visibility since you wanted to see the config.

Reply:
🙏